src/components/progress-flow/progress-flow-item/progress-flow-item.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+    Component,
+    Element,
+    Event,
+    EventEmitter,
+    h,
+    Prop,
+} from '@stencil/core';
+import { FlowItem } from '../progress-flow.types';
+
+/**
+ * @private
+ */
+@Component({
+    tag: 'limel-progress-flow-item',
+    shadow: false,
+    styleUrl: 'progress-flow-item.scss',
+})
+export class ProgressFlowItem {
+    @Element()
+    public element: HTMLLimelProgressFlowItemElement;
+
+    /**
+     * What flow item that should be rendered
+     */
+    @Prop()
+    public item: FlowItem = null;
+
+    /**
+     * True if the flow item should be disabled
+     */
+    @Prop()
+    public disabled = false;
+
+    /**
+     * True if the flow item should be readonly
+     */
+    @Prop()
+    public readonly = false;
+
+    /**
+     * Fired when clicking on the flow item
+     */
+    @Event()
+    public interact: EventEmitter<void>;
+
+    /**
+     * Icon displayed along with the text optionally
+     */
+    @Prop()
+    public icon: string;
+
+    constructor() {
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    public render() {
+        let secondaryText = this.item.secondaryText;
+        if (secondaryText) {
+            secondaryText = ' · ' + secondaryText;
+        }
+
+        const tooltip = this.item.text + secondaryText;
+
+        return [
+            <button
+                tabindex="0"
+                title={tooltip}
+                type="button"
+                class={{
+                    step: true,
+                    active: this.item?.selected,
+                    disabled: this.disabled || this.item?.disabled,
+                    readonly: this.readonly,
+                }}
+                onClick={this.handleClick}
+            >
+                {this.renderIcon()}
+                <span class="text">{this.item.text}</span>
+                <div class="divider" />
+            </button>,
+            this.renderSecondaryText(),
+        ];
+    }
+
+    private handleClick() {
```

Comment:
We could use an arrow function here instead like you did in handleFlowItemClick in the limel-progress-flow component. Then you don't have to bind it in the constructor. 🙂

Reply:
Awesome, please add a fixup